lib/jekyll/configuration.rb

```diff
@@ -175,7 +175,7 @@ def read_config_file(file)
       next_config
     rescue SystemCallError
       if @default_config_file ||= nil
-        Jekyll.logger.warn "Configuration file:", "none"
+        Jekyll.logger.warn "Configuration file:", "none (looking for '#{file}')"
```

Comment:
I agree with @parkr, we provide defaults for a reason, and that's because a config file is not necessary, the lack thereof is not an error, and shouldn't even be a warning if you were to ask me, it should simply be an informational message: "unable to find config file, using system defaults."

Warnings are meant for situations we recovered from, we didn't recover from this, we just didn't care and we've already got defaults that will suffice for a lot of generic sites being built.